tla/consensus/ccfraft.tla

```diff
@@ -462,22 +461,21 @@ PlausibleSucessorNodes(i) ==
 InitReconfigurationVars ==
     /\ reconfigurationCount = 0
     /\ removedFromConfiguration = {}
-    \* Note that CCF has a bootstrapping procedure to start a new network and to join new nodes to the network (see 
-    \* https://microsoft.github.io/CCF/main/operations/start_network.html). In both cases, a node has the current (see 
-    \* https://microsoft.github.io/CCF/main/operations/ledger_snapshot.html#join-or-recover-from-snapshot) or some stale configuration
-    \* such as the initial configuration. A node's configuration is *never* "empty", i.e., the equivalent of configuration[node] = {} here. 
-    \* For simplicity, the set of servers/nodes all have the same initial configuration at startup.
-    /\ \E s \in Servers:
-        configurations = [i \in Servers |-> 0 :> {s} ]
+    /\ \E startNode \in Servers:
```

Comment:
A helper function might be useful here, one which returns an init state given the initial configuration. This can be reused in the tracing and MC

Reply:
https://github.com/microsoft/CCF/pull/5828/commits/4c755e1775d56e5ba336ba99421557967113147a